api/app/Http/Controllers/Api/LeaderboardController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Services\Leaderboard\Leaderboard;
+
+class LeaderboardController extends Controller
+{
+    private $leaderboardService;
+
+    public function __construct(Leaderboard $leaderboardService)
+    {
+        $this->leaderboardService = $leaderboardService;
+    }
+
+    public function index()
+    {
+        return $this->leaderboardService->forAllUsers();
+    }
+}
```

Comment:
Maybe add function to get leaderboard around specific user? Or around a specific position? For account purpose to see your 'direct competitors'?